examples/sn-dms-demo/cypress/integration/user/logins.spec.ts

```diff
@@ -0,0 +1,43 @@
+context('The login page', () => {
+  beforeEach(() => {
+    cy.visit('/#/login')
+  })
+
+  it('requires email', () => {
+    cy.get('form')
+      .contains('Login')
+      .click()
+    cy.contains('Please provide a valid e-mail address!').should('exist')
```

Comment:
Maybe we should discuss this type of tests because I think it's not the best way to test by text messages. A minor text change and tests will fail or the apps language is changed to another language and the tests will fail. 😿

Reply:
I've replaced all the strings that were in the resources.